### PR TITLE
Remove some of our middleware classes from stacktraces and logging

### PR DESCRIPTION
**Pull request by sadleb**
### Description
These classes are in the stack for everything and can be a red herring
when you see them.

Note: set `ENV["BACKTRACE"]=1` if you run into an issue were you do want the silenced classes in the traces and logs

### Motivation and Context
These annoy me.

#### postgresql_adapter.rb_rb_BEFORE
image

#### postgresql_adapter.rb_rb_AFTER
image

#### lti_rise360_proxy_BEFORE.png
image

#### lti_rise360_proxy_AFTER.png
image

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist:

- [ ] My code follows the code style of this project.
- [ ] I have added neccessary labels
- [ ] My PR title follows the convention in the guide
- [ ] I have assigned myself to the PR
- [ ] I have requested reviews from team members
- [ ] I have created a task for reviewers on PM tool
- [ ] All new and existing tests passed.

### CODE REVIEW

File: config/initializers/backtrace_silencers.rb [hunk 1]
```diff
@@ -3,6 +3,13 @@
 # You can add backtrace silencers for libraries that you're using but don't wish to see in your backtraces.
 # Rails.backtrace_cleaner.add_silencer { |line| /my_noisy_library/.match?(line) }
 
+Rails.backtrace_cleaner.add_silencer { |line|
+  /core_ext\/postgresql_adapter/.match?(line) || \
+  /honeycomb_sentry_integration/.match?(line) || \
+  /honeycomb_trace_propagation/.match?(line) || \
+  /lti_rise360_proxy/.match?(line)
```
Comment (rshipp) [inline config/initializers/backtrace_silencers.rb:10]: can you explain the proxy one more? i see the example screenshot but I don't understand in what scenario real errors would be incorrectly pointing to the proxy code

Comment (sadleb) [reply config/initializers/backtrace_silencers.rb:10]: Every request goes through the proxy. It’s middleware as part of Rails routing. For all requests but the ones where it actually proxies the request it’s a NOOP. Any exception that happens as part of a request will have that in the stack trace. There is a 0.001% chance we’d want to look in that code to figure out what is going wrong, so it’s just a distraction if you don’t remember that it’s just always there.

Comment (sadleb) [reply config/initializers/backtrace_silencers.rb:10]: In other words, Rails also has a ton of stuff that happens before it hits the controller but we don’t want to see it in the stacktraces bc we’re relying on the Rails routing logic not to have a bug.

Comment (rshipp) [reply config/initializers/backtrace_silencers.rb:10]: oh ok, i didn't realize everything went through the proxy code, that's annoying.

**Review by rshipp — APPROVED** [review]
didn't know you could do this

**Pull request merged**